python/dynamic programming/problem_0198_house_robber.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+
+class Solution:
+    @staticmethod
+    def rob(nums):
```

Comment:
Consider renaming nums to something more meaningful.

Reply:
we don't need to rename Leetcode input parameters

class Solution:
    def rob(self, nums: List[int]) -> int: